src/containers/MyNdla/components/NavigationLink.tsx

```diff
@@ -38,6 +38,23 @@ const ShortText = styled("span", {
   },
 });
 
+const IconWrapper = styled("span", {
+  base: {
+    "& svg": {
```

Comment:
Trengte dette for å overskrive stylingen fra MyNdlaButton:/

Reply:
Skjønner ikke helt, kan du ikke bare style ikonet direkte?

Reply:
Nei, ser dårlig ut😅  Etersom button har size medium vil stylingen av svg der få høyere spesifisitet